src/bin/flamegraph.rs

```diff
@@ -2,18 +2,31 @@ use std::fs::File;
 use std::io::{self, BufReader};
 use structopt::StructOpt;
 
-use inferno::flamegraph::{self, Options};
+use inferno::flamegraph::{self, Options, color::Palette};
 
 #[derive(Debug, StructOpt)]
 #[structopt(name = "inferno-flamegraph", author = "")]
 struct Opt {
     /// collapsed perf output file, or STDIN if not specified
     infile: Option<String>,
+    /// set color palette. choices are: hot, mem, io, wakeup, chain, java, js, perl, red, green, blue, aqua, yellow, purple, orange
+    #[structopt(long = "colors", default_value = "hot")]
```

Comment:
Let's also use [`possible_values`](https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.possible_values) here:
```rust
#[structopt(..., raw(possible_values = r#"&["hot","mem","io","wakeup","chain","java","js","perl","red","green","blue","aqua","yellow","purple","orange"]"#)]
```
That way the user will get a helpful error if they try to use an unrecognized value.

Also, maybe we should not list `chain` since it's not truly supported?

Reply:
I'd also like us to add `short = "c"` since I think this is a pretty common option.